insights/parsers/yum_updateinfo.py

```diff
@@ -0,0 +1,31 @@
+"""
+UpdateInfo - command ``yum updateinfo list -C``
+===============================================
+Provides a list of available advisories
+"""
+from insights import CommandParser, parser
+from insights.parsers import parse_delimited_table, SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.yum_updateinfo)
+class YumUpdateinfo(CommandParser):
+    """
+    Class for parsing the output of yum updateinfo list.
+
+    Output:
+        >>> [{ "advisory": "RHSA-2020:5206", "type":"Sec./Moderate", "package":"kernel-5.9.12-100.el8.x86_64"}]
+    """
+
+    def parse_content(self, content):
+        """Parse the command output"""
+        with_header = ['advisory type package'] + content
+        table = parse_delimited_table(with_header)
+        if not table:
+            raise SkipException('No data.')
+        self._items = table
+
+    @property
+    def items(self):
+        """Get advisory items"""
```

Comment:
Since this is the "attribute" for this parser you should include the return type and possibly an example of the return value.  See this [property method](https://github.com/RedHatInsights/insights-core/blob/863624a95054d7ee2eac2a4553b616c1ea569e18/insights/parsers/yum.py#L176) as an example.